doc/api/n-api.md

```diff
@@ -2889,25 +2889,24 @@ Returns `napi_ok` if the API succeeded.
 This method is used within a callback function to retrieve details about the
 call like the arguments and the `this` pointer from a given callback info.
 
-### *napi_is_construct_call*
+### *napi_get_new_target*
 <!-- YAML
-added: v8.0.0
+added: v9.0.0
```

Comment:
Use `REPLACEME` instead of a specific version. Whoever does a release will then update it to the correct version. (This will be in some later 8.x release.)